### PR TITLE
Blog app: add devise

### PR DESCRIPTION
**Pull request by luccamicale**
## Blog app: add devise 💯 

##https://github.com/luccamicale/Blog_Rails 👍 

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] Build a web app that requires the user to log in.
- [x] Use devise gem for authentication.
- [x] Best practices in Rails and linters 🥇 🔢 💯 👍

### CODE REVIEW

**Comment by luccamicale** [comment]
I follow the all requirements, work with devise and then add the styles. My project it is complete, should be approved!!
👍 💯 🔢 🥇

File: app/models/user.rb [hunk 1]
```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```
Comment (t-yanick) [inline app/models/user.rb]: - [x] As per the project requirements, the application is supposed to ask for confirmation of email. For this functionality to work, `:confirmable` is supposed to be enabled which is not the case right now, Please kindly enable :`confirmable.`
```
devise :database_authenticatable, :registerable,
				:recoverable, :rememberable, :validatable, :confirmable
```

File: README.md [hunk 2]
```diff
@@ -144,11 +144,10 @@ You can deploy this project using:
...
 ## 👥 Authors <a name="authors"></a>
 
-👤 **Odette Gleen**
+👤 **Lucas Bermudez**
 
-- GitHub: [@Gleennkar](https://github.com/Gleennkar)
-- Twitter: [@gleeny24](https://twitter.com/twitterhandle)
-- LinkedIn: [@gleeny](https://www.linkedin.com/in/gleeny-nkar-aa3917182)
+- GitHub: [@Oklukeok](https://github.com/Oklukeok)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/lucas-bermudez)
```
Comment (t-yanick) [inline README.md:153]: - [x] Great job adding devise. In your `ReadMe`, under the `Install` section, you say `No Install Needed, `but I think I had to bundle install, create database and migrate it before i could start the server locally. Please kindly all these steps to enable a user run or start your app with ease.

**Review by t-yanick — CHANGES_REQUESTED** [review]
Hi @luccamicale and @Oklukeok , 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Great job installing and setting up Devise
:heavy_check_mark: Collaborative work 😎😎
:heavy_check_mark: Gitflow followed

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @luccamicale, 

### Good job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark:  Clean and Organized code
:heavy_check_mark: Descriptive Readme file

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack  using this handle @mihndim.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**